tests/tests_pytorch/utilities/test_model_summary.py

```diff
@@ -270,7 +271,7 @@ def test_summary_with_scripted_modules(max_depth):
         ([], UNKNOWN_SIZE),
         ((1, 2, 3), [UNKNOWN_SIZE] * 3),
         (torch.tensor(0), UNKNOWN_SIZE),
-        ({"tensor": torch.zeros(1, 2, 3)}, UNKNOWN_SIZE),
+        ({"tensor": torch.zeros(1, 2, 3)}, [1, 2, 3]),
```

Comment:
could we add the following:

```suggestion
        ({"tensor": torch.zeros(1, 2, 3)}, [1, 2, 3]),
        ({"tensor0": torch.zeros(1, 2, 3), "tensor1": torch.zeros(4, 5, 6)}, [[1, 2, 3], [4, 5, 6]]),

```

This extends the test case to multiple kwarg entries, which is what the new implementation should support now :)